mysite/blog/Templates/Blog/portfolio_index_page.html

```diff
@@ -0,0 +1,22 @@
+{% extends "base.html" %}
+{% load wagtailcore_tags %}
+{% load static %}
+
+{% block extra_head %}
+<link rel="stylesheet" href="{% static 'blog/css/portfolio_styles.css' %}">
+{% endblock extra_head %}
+
+{% block content %}
+    <h1>{{ page.title }}</h1>
+    {{ page.intro|richtext }}
+
+    {% for item in portfolio_items %}
+        <div class="portfolio-item">
+            <h2><a href="{% pageurl item %}">{{ item.title }}</a></h2>
+            <p> {{ item.description|truncatechars_html:200 }}</p>
```

Comment:
You may also need to pipe this to the richtext filter. Remove the truncate for now so you can test the richtext.
```suggestion
            <p> {{ item.description|richtext }}</p>
```